packages/ipfs-http-client/src/add-all.js

```diff
@@ -6,10 +6,14 @@ const configure = require('./lib/configure')
 const multipartRequest = require('./lib/multipart-request')
 const toUrlSearchParams = require('./lib/to-url-search-params')
 const anySignal = require('any-signal')
-const AbortController = require('abort-controller')
+const { AbortController } = require('abort-controller')
 
 module.exports = configure((api) => {
-  return async function * addAll (input, options = {}) {
+  // eslint-disable-next-line valid-jsdoc
```

Comment:
Why are we knowingly adding invalid js doc comments?  Is the type here (and all the following instances) something that cannot be expressed?

Reply:
`valid-jsdoc` requires `@param` definitions for functions even `@type` presents

Reply:
BTW, ESLint's `valid-jsdoc` has been [deprecated](https://eslint.org/blog/2018/11/jsdoc-end-of-life) for long time. We should replace this.